packages/xstate-vue/package.json

```diff
@@ -25,15 +25,17 @@
   "files": [
     "lib/**/*.js",
     "lib/**/*.d.ts",
-    "dist/**/*.js"
+    "dist/**/*.js",
+    "es/**/*.js",
```

Comment:
the ESM build should be added correctly using `"module"` package.json key so people don't have to worry about picking it manually

Reply:
True! Forgot to add it. Added:
`"module": "es/index.js"` 👍